<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finishOrder</class>
 <widget class="QMainWindow" name="finishOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#finishOrder{
background-color: qlineargradient(spread:pad, x1:0, y1:0.449, x2:0, y2:1, stop:0.60199 rgba(255, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="topside" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="leftWidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#leftWidget{
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 2px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="orderHead" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border-bottom: 2px solid black;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QFrame" name="orderlbFrm">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="orderLabel">
                  <property name="styleSheet">
                   <string notr="true">border: none;
color: rgb(0, 0, 0);
font: 26pt &quot;Arial&quot;;</string>
                  </property>
                  <property name="text">
                   <string>Order: </string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="ordernumfrm">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="orderNumLabel">
                  <property name="styleSheet">
                   <string notr="true">border: none;
color: rgb(0, 0, 0);
font: 26pt &quot;Arial&quot;;</string>
                  </property>
                  <property name="text">
                   <string>00</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="orderLstWid" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QTableWidget" name="orderTableWidget">
               <property name="styleSheet">
                <string notr="true">QHeaderView::section {
    background-color: rgb(213, 248, 255);
    padding: 2px;
    border: 1px solid;

	font: 18pt &quot;Arial&quot;;
}</string>
               </property>
               <column>
                <property name="text">
                 <string>Menu Item </string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Quantity</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Notes</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Price</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="rightWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QFrame" name="nameFrm">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="nameLabel">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 0, 0);
font: 26pt &quot;Forte&quot;;</string>
               </property>
               <property name="text">
                <string>Name:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignBottom">
              <widget class="QLabel" name="totalLabel">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 0, 0);
font: 26pt &quot;Forte&quot;;</string>
               </property>
               <property name="text">
                <string>Total:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QFrame" name="inputNameFrm">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="inputNameLineEdit">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);
font: 26pt &quot;Arial&quot;;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="inputTotalFrm">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item alignment="Qt::AlignBottom">
                 <widget class="QLabel" name="displayTotal">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);
font: 26pt &quot;Arial&quot;;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="backBtn">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(111, 210, 255);
font: 75 26pt &quot;Arial&quot;;
border-style: outset;
border-width: 5px;
border-color: rgb(21, 76, 152);</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>800</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="submitOrderBtn">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(111, 210, 255);
font: 75 26pt &quot;Arial&quot;;
border-style: outset;
border-width: 5px;
border-color: rgb(21, 76, 152);</string>
         </property>
         <property name="text">
          <string>Submit Order</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
